gnome-initial-setup/gis-driver.c

```diff
@@ -80,12 +81,37 @@ struct _GisDriverPrivate {
   gchar *personality;
   gchar *lang_id;
 
+  gboolean is_live_session;
+
   GisDriverMode mode;
 };
 typedef struct _GisDriverPrivate GisDriverPrivate;
 
 G_DEFINE_TYPE_WITH_PRIVATE(GisDriver, gis_driver, GTK_TYPE_APPLICATION)
 
+static gboolean
+running_live_session (void)
```

Comment:
I wonder if it would be cleaner (and easier to test) to add a `--skip-pages` command-line argument. Actually there's already a magic `/tmp/skip_pages_file` which could be used to do this… not a big deal I suppose.